### ISSUE TITLE
Collect lists of files that may exist for processing

### ISSUE THREAD

**Issue by gonuke** [svalinn/ALARA#52]
Determine which files of each type are present and create a list of such files for processing.

**Comment by eitan-weinstein** [comment]
Yep, this is covered in `tendl_processing.py` in the `search_for_files()` function (lines 28-62).
